### ISSUE TITLE
[mesheryctl] `mesheryctl system context view <context-name>` 

### ISSUE THREAD

**Issue by navendu-pottekkat** [layer5io/meshery#2830]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

Related issue #2829 

#### Current Behavior

Users can view the current-context as shown below-

```
❯ mesheryctl system context view
Current context: local

endpoint: http://localhost:9081
token: Default
platform: kubernetes
adapters:
- meshery-istio
channel: stable
version: latest
```

#### Desired Behavior

Users should also be able to view contexts that are not the `current-context` by their names-

```
❯ mesheryctl system context view bob
Current context: bob

endpoint: http://localhost:9081
token: Default
platform: docker
adapters:
- meshery-linkerd
channel: stable
version: latest
```

---

#### Contributor Resources

- [mesheryctl Contributing Guide](https://github.com/layer5io/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)

**Comment by samirparhi-dev** [comment]
@navendu-pottekkat  : let me take this up.

**Comment by navendu-pottekkat** [comment]
Fixed in #2880
